bitcoin/src/blockdata/transaction.rs

```diff
@@ -230,8 +230,6 @@ impl TxIn {
     /// Returns the input base weight.
     ///
     /// Base weight excludes the witness and script.
-    // to be used in up-coming commit to add effective_value()
-    #[allow(dead_code)]
```

Comment:
Oh, I now noticed this. It'd be better to squash the commits. There's no point in adding an unused constant in a separate commit.

Reply:
Agreed

Reply:
thanks, done.